src/PrestaShopBundle/Resources/views/Admin/AdvancedParameters/system_information.html.twig

```diff
@@ -74,7 +74,7 @@
 
         <div class="card">
             <h3 class="card-header">
-                <i class="material-icons">info_outline</i> {{ 'Database information'|trans }}
+                <i class="material-icons">info_outline</i> {{ 'Database information'|trans({}, 'messages') }}
```

Comment:
As we will reach 1.7.3 and thus can break some translations, can't we use another domain than this "messages"?

Reply:
ok, but which one(s) ?

Reply:
I guess it should exist in `Admin.Advparameters.Feature`. If not, this will be fixed on Crowdin later.

Reply:
I forgot to change this key, sorry